packages/experience-components/vite.config.ts

```diff
@@ -0,0 +1,38 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import dts from 'vite-plugin-dts';
+import path from 'path';
+import { libInjectCss } from 'vite-plugin-lib-inject-css';
+import svgr from 'vite-plugin-svgr';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), libInjectCss(), dts({ rollupTypes: true }), svgr()],
```

Comment:
can we just use the source code for now? i don't think we need to compile frontend components at the moment since vite can check and transpile the source

Reply:
Remove the vite build for this project.